src/components/NotificationWindow.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import * as s from './notificationWindow.scss';
+import classnames from 'classnames';
+
+interface IErrorWindow {
+    notifications: object[];
+    hideError: Function;
+}
+
+class ErrorWindow extends React.Component<IErrorWindow> {
+
+    public onErrorWindowClick = (message: string) => {
+        this.props.hideError(message);
+    }
+
+    getColorClass = (type: string) => {
+        switch (type) {
+        case 'error': {
```

Comment:
I would suggest migrating to using enums instead of strings for passing error types.